frontend/src/pages/NewRunV2.tsx

```diff
@@ -812,7 +840,7 @@ function PipelineVersionSelector(props: PipelineVersionSelectorProps) {
       <Input
         value={props.pipelineVersionName}
         required={true}
-        label='Pipeline Version'
+        label={props.disabled ? 'Using latest pipeline version' : 'Pipeline Version *'}
```

Comment:
This prevents the double asterisk from showing up.
```suggestion
        label={props.disabled ? 'Using latest pipeline version' : 'Pipeline Version'}
```